draft-ietf-scitt-architecture.md

```diff
@@ -619,6 +619,40 @@ For example, consider a Transparency Service implemented using a set of replicas
 Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
 This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
 
+### Configuration
+
+The Transparency Service records its configuration in the Append-Only Log using Transparent Statements with distinguished media type `application/scitt-configuration`.
+
+The registration policy for statements with the media type suffix (`+<format>` is implementation-specific.
+The implementation SHOULD document them, for example defining the Issuers authorized to register configuration Signed Statements.
+
+The Transparency Service is configured by the last Transparent Statement of this type.
+The Transparency Service MUST register a Signed Statement that defines its initial configuration before registering any other Signed Statement.
+The Transparency Service MAY register an additional Signed Statement that updates its configuration.
+
+The Transparency Service provides an endpoint that returns the Transparency Statement that describes its current configuration.
+
+The configuration `reg_info` SHOULD include a secure version number and a timestamp.
```

Comment:
To avoid rollbacks, we SHOULD likely make this a MUST, specific to configuration.

Reply:
Does anyone have an opinion on changing this PR, or making a separate? In the spirt of merging quickly, and iterating, I'm fine deferring to another PR.